package.json

```diff
@@ -4,24 +4,28 @@
   "description": "",
   "main": "./src/server.js",
   "scripts": {
+    "test": "mocha",
     "start": "nodemon",
-    "prettier-check": "prettier --check src/**/*.js",
-    "prettier-format": "prettier --write src/**/*.js",
+    "prettier-check": "prettier --check ./**/*.js",
+    "prettier-format": "prettier --write ./**/*.js",
     "eslint": "eslint ."
```

Comment:
Okazuje się że matchowanie `**` działa inaczej na linuxie niż na windowsie, tzn. na windowsie matchuje podfoldery na dowolnej głębokości a na linuxie jest równoważne `*` i na linuxie nie da się zmatchować `/` globem inaczej niż napisać go explicit.
Okazuje się również, że `prettier --check .` sprawdzi wszystko w folderze `.` i jego wszsystkich podfolderach na dowolnej głębokości (pomijając `node_modules`) oraz całkiem dobrze sobie radzi z foramtowaniem innych plików niż `.js`.
Zatem dobrym rozwiązaniem jest zmianan tych dwóch linijek na:
`prettier --check .`
`prettier --write .`

Reply:
jak się poprawi to krzyczy na dwa twoje pliki (oraz na 3 moje) ale prettier-write je ładnie poprawia